packages/server/graphql/intranetSchema/mutations/addNewFeature.ts

```diff
@@ -43,14 +45,7 @@ const addNewFeature = {
       db.writeTable('User', {newFeatureId})
     ])
 
-    const onlineUserIds = await r
-      .table('User')
-      .filter((user) =>
-        user('connectedSockets')
-          .count()
-          .ge(1)
-      )('id')
-      .run()
+    const onlineUserIds = await redis.smembers('onlineUserIds')
```

Comment:
The original proposal in Notion was to include User Presence and online users in a team in Redis but I thought it'd also be helpful to store all online users so that we can access them in `O(N)` here. 

Alternate approaches I considered were:
1. Getting all of the `teamIds` from RethinkDb and then getting all of the online users from the `team:${teamId}` but this hits both the db and Redis
2. Use [KEYS](https://redis.io/commands/keys) to get all online user keys matching a pattern, i.e. `presence`. This would be a bit of a hack and Redis don't advise it in production. 

Taking up a bit more space in Redis seems like the right approach to me, particularly as we might want to access all `onlineUserIds` elsewhere in the app in the future.